### DISCUSSION TITLE
不是内部或外部命令，也不是可运行的程序

### DISCUSSION THREAD

**Discussion by Cjhyyd** [Stability-AI/StableSwarmUI#421] [Q&A]
![微信截图_20240620151308](https://github.com/Stability-AI/StableSwarmUI/assets/143602316/1c9b318e-3120-45da-aed4-a4e53f6b3391)

**Comment by mcmonkey4eva** [comment]
I'm sorry, I don't speak Chinese, but this looks like a common installer error - you most likely just need to open the folder `StableSwarmUI` and double-click the `launch-windows.bat` file

**Reply by Cjhyyd** [reply]
I placed it on GitHub alone, but it couldn't start either. It flashed by.

**Comment by Cjhyyd** [comment]
Title: Not an internal or external command, nor a runnable program
